notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<p>Hi Sierra, it's important that we keep this line of code within the cell to ensure that the following cells can run without any issues. Otherwise, the output file 'crds_output_1.txt' will only contain 2 lines for 'env RDS_SERVER_URL' and 'env CRDS_PATH', which will cause a KeyError when we attempt to print the first and last 5 lines of the output in the next cell.</p><p> </p><p>Please add the following to the code cell:</p><p> </p><pre class="ql-syntax" spellcheck="false">!crds bestrefs --files data/*raw*.fits&nbsp;--sync-references=2 --update-bestrefs --new-context 'hst_1071.pmap'&nbsp;
</pre><p> </p><p>Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>